lib/stomp_handler.dart

```diff
@@ -46,8 +51,13 @@ class StompHandler {
       channel = await platform.connect(config);
       channel.stream.listen(_onData, onError: _onError, onDone: _onDone);
       _connectToStomp();
-    } on WebSocketChannelException catch (err) {
-      _onError(err);
+    } on WebSocketChannelException catch (err) {      
+      if (config.reconnectDelay == 0) {
+        _onError(err);
+      } else {
+        config.onDebugMessage('Connection web socket error...reconnecting');
+        _onDone();
+      }
     } on TimeoutException catch (err) {
```

Comment:
I have seen the issue you opened. Could you maybe revert those few lines and open either a separate issue/PR for that? Keeps it a bit more organized.

Reply:
ok i revert and submit new PR